contracts/contracts/RequestManager.sol

```diff
@@ -27,6 +27,7 @@ contract RequestManager is Ownable {
         uint192 activeClaims;
         uint256 validUntil;
         uint256 lpFee;
```

Comment:
Maybe rename `lpFee` to `lpFeeToken`, theoretically speaking, the total lp fees include both the gas fee in eth and the service fee in token.

Reply:
That's a good idea, will do.

Reply:
Done.